app/src/main/java/org/gdg/frisbee/android/pulse/PulseAdapter.java

```diff
@@ -39,13 +40,18 @@
 
     private LayoutInflater mInflater;
     private ArrayList<Map.Entry<String, PulseEntry>> mPulse;
-    private int[] mPosition;
+    private int[] mPositions;
```

Comment:
When I disabled the ViewPager's aggressive caching, it solved the numbering problem for changing countries/cities. But it didn't solve when orientation change occurs. When orientation change occur in a page, the list scroll position is kept but the computed Positions were not kept. We may solve the problem with changing the algorithm to compute positions but it requires more work. I've just added kept the positions computed.